modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -828,8 +829,8 @@ def process_draft_appointment(referral_id, referral_consult_id)
                                  address: referral.treating_facility_address)
         return { success: false, json: provider_not_found_error, status: :not_found } unless provider&.id
 
-        slots = fetch_provider_slots(referral, provider)
         draft = eps_appointment_service.create_draft_appointment(referral_id:)
+        slots = fetch_provider_slots(referral, provider, draft.id)
```

Comment:
just making a note here that the order of operations had to change in order to send the `appointmentId` in the slots request